src/components/deposit/Congratulations.js

```diff
@@ -27,22 +36,25 @@ const Congratulations = ({ depositAddress, lotInSatoshis, signerFeeInSatoshis })
       </div>
       <hr />
       <div className="description">
+        {
+          depositAddress && depositAddress.length > 0
+          ? <>
+              <div className="deposit-address-label">Deposit Address:</div>
+              <CopyAddressField address={depositAddress} />
+            </>
+          : ''
+        }
         <div className="description-content">
           You are now the proud beneficiary of {lotInTbtc.toNumber()} TBTC
         </div>
         <div className="bond-duration">
           Bond duration: 6 months
         </div>
-        {/* TODO: Update to use CopyInputField */}
-        {
-          depositAddress && depositAddress.length > 0
-          ? <div>
-              <br />
-              <h3>TDT ID:</h3>
-              { depositAddress }
-            </div>
-          : ''
-        }
+        <button className="black">
+          <a href={
+            `https://${chainId === 3 ? 'ropsten.' : ''}etherscan.io/token/${depositAddress}`
```

Comment:
Would be kind of nice to abstract this somewhere at some point, though admittedly we're only using it in one place right now.

Reply:
I was thinking the same thing just to make it easier to read the code, but didn't pull the trigger bc it's only once instance.

Reply:
There is actually another instance on the redemption congrats page! Addressed in b1626bd